src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -52,9 +52,38 @@ const BoundedCacheInitialState: BoundedCache = {
     evictionStrategy: EvictionStrategy.REMOVE,
 }
 
+const IndexWriterInitialState: IndexWriter = {
+    commitInterval: -1,
+    lowLevelTrace: false,
+    maxBufferedEntries: -1,
+    queueCount: -1,
+    queueSize: -1,
+    ramBufferSize: -1,
+    threadPoolSize: -1,
+}
+
+const IndexMergeInitialState: IndexMerge = {
+    calibrateByDeletes: false,
+    factor: -1,
+    maxEntries: -1,
```

Comment:
Needs to be reviewed. I don't know if we want to let all this to be configured this way

Reply:
@oraNod as [suggested](https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/Cache.20Wizard/near/279398697) , added the options for tuning indexing, let us know if all the options are necessary or we can remove some.

Reply:
Just to be clear with my proposal above the wizard would have this navigation:

- Basic options 
- Infinispan capabilities 
- Cache tuning options

Reply:
Right now I think the indexing section is pretty spot on, we just need to move all the tuning parameters out of there.